asmrun/arm.S

```diff
@@ -27,7 +27,7 @@
         cmp     \reg, #0
         beq     \lbl
         .endm
-#elif defined(SYS_linux_eabihf)
+#elif (defined(SYS_linux_eabihf) || (defined(SYS_netbsd)) && defined(MODEL_armv7))
```

Comment:
It looks like the case SYS_netbsd && MODEL_armv6 is not covered.  I suggest refactoring starting at line 20:
```
    #if defined(SYS_linux_eabihf) || defined(SYS_netbsd)
    #if defined(MODEL_armv6)
    ....
    #else  
    ....
    #endif 
    #elif defined(SYS_linux_abi)
```